tests/invalid_models_tests/test_models.py

```diff
@@ -706,6 +706,26 @@ class Test__Model(models.Model):
             )
         ])
 
+    def test_property_name_foreign_key_id_accessor_clashes(self):
+        class ModelOne(models.Model):
+            pass
+
+        class ModelTwo(models.Model):
+            one = models.ForeignKey(ModelOne, on_delete=models.CASCADE)
```

Comment:
Drop `on_delete=` for consistency with other code.

I think the ForeignKey could be to 'self' to avoid the need for the other model.